lib/kb_SPAdes/kb_SPAdesImpl.py

```diff
@@ -421,23 +484,20 @@ def run_SPAdes(self, ctx, params):
 
         self.log('Got reads data from converter:\n' + pformat(reads))
 
-        self.check_reads(params, reads, reftoname)
+        phred_type = self.check_reads(params, reads, reftoname)
 
         reads_data = []
-        for ref in sorted(reads):
+        for ref in reads:
             reads_name = reftoname[ref]
             f = reads[ref]['files']
-            if f['type'] == 'single':
```

Comment:
This check was moved into check reads, otherwise a different error was hit, as it could not download the files.